scripts/processors/cams_processor.py

```
@@ -0,0 +1,343 @@
+            
+        # Aggregate to H3 hexagons with hourly resolution
+        if 'h3_index_res8' in result.columns and 'timestamp' in result.columns:
+            logger.info("Aggregating to H3 hexagons...")
+            
+            # Round timestamps to hour
+            result['timestamp_hour'] = pd.to_datetime(result['timestamp']).dt.floor('h')
+            
+            # Group by hexagon and hour
+            agg_dict = {}
+            for col in result.columns:
+                if col.endswith('_ugm3') or col.endswith('_ppm'):
```

Ruff (0.12.2)

Call `endswith` once with a `tuple`

Merge into a single `endswith` call

(PIE810)